aidants_connect_common/static/css/common.css

```
@@ -0,0 +1,20 @@
+@media  (min-width: 48em) {
+  .text-md-right {
+    text-align: right;
+  }
+}
+.fr-grid-row--reverse {
+  flex-direction: row-reverse;
+}
+.fr-text-blue-france {
+  color: var(--artwork-major-blue-france);
+}
+.fr-text-black {
+  color: black;
+}
```

```suggestion
.fr-text-default {
  color: var(--text-default-grey);
}
```

Pour permettre l'utilisation potentielle d'un thème sombre.

À mettre dans [la section utils de `aidants_connect_common/static/scss/main.scss`](https://github.com/betagouv/Aidants_Connect/blob/main/aidants_connect_common/static/scss/main.scss#L87). C'est ce fichier qui donne le design commun. Tu peux utiliser `python manage.py scss --watch` pour compiler le SCSS.

---

je pensais qu'on voulait se débarrasser du SCSS ?

---

Quand c'est pas nécessaire, oui. Mais dans `main.scss`, j'utilise des fonctions Sass.